src/select-field.jsx

```diff
@@ -141,6 +141,8 @@ const SelectField = React.createClass({
       hintText,
       fullWidth,
       errorText,
+      onFocus,
```

Comment:
I'm wondering. What is going to be the side effect?
DropDownMenu handle the onFocus property `https://github.com/callemall/material-ui/blob/master/src/drop-down-menu.jsx#L228`